homeassistant/components/airzone_cloud/climate.py

```diff
@@ -386,4 +387,6 @@ async def async_set_hvac_mode(self, hvac_mode: HVACMode) -> None:
         await self._async_update_params(params)
 
         if slave_raise:
-            raise HomeAssistantError(f"Mode can't be changed on slave zone {self.name}")
+            raise HomeAssistantError(
+                f"Mode can't be changed on slave zone {self.get_airzone_value(AZD_NAME)}"
```

Comment:
Another option which I think is more used is `self.entity_id` given that a user can rename their devices etc.

Reply:
> Another option which I think is more used is `self.entity_id` given that a user can rename their devices etc.

Done in 1cfc8886e5f840d6eb2df9981cc856cb109372d9
Thanks!